collection-scripts/gather_etcd

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```

Comment:
This probably needs guards like:

```sh
set -o nounset
set -o errexit
set -o pipefail
```

so that it doesn't blindly grind on if a step like calculating `RUNNING_ETCD_POD` fails.

Reply:
I am ok with `nounset` and `pipefail`. `errexit` would only be appropriate for `RUNNING_ETCD_POD` and `ETCDCTL_ENDPOINTS` calculation, because if one of the `ocp4etcdctl` commands return a non-zero exit code, the other ones should be tried.

It would be good to evaluate if any of the other files needs these guards, because none of them use them.